layers/+lang/plantuml/packages.el

```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- plantuml layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Robert O'Connor <robby.oconnor@gmail.com>
+;; URL: https://github.com/robbyoconnor
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+
+(defconst plantuml-packages
+  '(puml-mode))
+
+(defun plantuml/init-puml-mode ()
+  (use-package puml-mode
+    :defer t
+    :mode ("\\.puml" . puml-mode) ("\\.plantuml" . puml-mode)
```

Comment:
Hi,

I'd like to add you to my professional network on LinkedIn.

- Robert

Confirm that you know Robert: https://www.linkedin.com/e/v2?e=0-ini67oeq-lk&t=ssuw&tracking=eml-guest-invite-cta&ek=invite_guest&sharedKey=0Ay2mjaX&invitationId=6130896983013023745








You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=0-ini67oeq-lk&t=lun&midToken=AQFmrFVhGEd36Q&ek=invite_guest&loid=AQHquEzEIjyHCQAAAVRVScFYqvvxSXNPBYFqEmPzQ-D8eLFEhWV_aQy6RtnD-mmenizQVzSgVHpGv39NyfZMi35Fc0NW8PFwxMZXgLm9q4tOsDL99dbHGewjr5ALdQd59cLeX78NE4kcO1lXB4Apum8zUrwgnvOfRyfDGibHxD2z0wLhmlHUJLjl4jdb_PG-8-_R&eid=0-ini67oeq-lk


This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=0-ini67oeq-lk&a=customerServiceUrl&ek=invite_guest

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.